src/modules/Modules.tsx

```diff
@@ -10,11 +10,12 @@ import ModulesRouter from './ModulesRouter';
 const Modules = () => {
   return (
     <>
-      <AlertsManager />
       <AuthProvider>
         <PatternsProvider>
           <TechnologiesProvider>
-            <ModulesRouter />
+            <AlertsProvider>
```

Comment:
What if i would like to display Alert in AuthProvider ? This should be placed higher

Reply:
done